### ISSUE TITLE
Control animations clipped inside card

### ISSUE THREAD

**Issue by Awsmolak** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2446]
When a control with an animation is placed near the edge of the card, animations that can extend beyond the border of the card content can be clipped.
![image](https://user-images.githubusercontent.com/15197249/135141256-de45056b-b798-410d-ae1a-128ea7dc2fcc.png)

**Comment by Keboo** [comment]
I suspect the fix for this issue is likely to just remove the Padding template binding from PART_ClipBorder [here](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Card.xaml#L28)

**Comment by Erapchu** [comment]
Please, can you try to set Panel.ZIndex to 1 for toggle button?

**Comment by Erapchu** [comment]
But on my side this combinations of controls is ok.

![image](https://user-images.githubusercontent.com/37536541/135476952-3d4a8e1f-4a90-4cc9-ae00-3739a89a1568.png)

**Comment by Awsmolak** [comment]
Panel.ZIndex="1" didn't work.
But after testing I remembered that I created an app wide default style for materialdesign:card to set a default margin and padding! So it looks like the padding was cutting off the animations. I guess I will need to go in and set the margins for all of the user controls hosted in cards instead of setting padding for the card. Interesting that animations can display outside margins but not paddings.
